autogluon/setup.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+import os
+import shutil
+import subprocess
+import codecs
+import os.path
+
+from setuptools import setup, find_packages, find_namespace_packages
+
+cwd = os.path.dirname(os.path.abspath(__file__))
+
+with open(os.path.join('..', 'VERSION')) as version_file:
+    version = version_file.read().strip()
+
+"""
+To release a new stable version on PyPi, simply tag the release on github, and the Github CI will automatically publish 
+a new stable version to PyPi using the configurations in .github/workflows/pypi_release.yml . 
+You need to increase the version number after stable release, so that the nightly pypi can work properly.
+"""
+try:
+    if not os.getenv('RELEASE'):
+        from datetime import date
+        today = date.today()
+        day = today.strftime("b%Y%m%d")
+        version += day
+except Exception:
+    pass
+
+def create_version_file():
+    global version, cwd
+    print('-- Building version ' + version)
+    version_path = os.path.join(cwd, 'src', 'autogluon', 'version.py')
+    with open(version_path, 'w') as f:
+        f.write('"""This is autogluon version file."""\n')
+        f.write("__version__ = '{}'\n".format(version))
+
+
+long_description = open(os.path.join('..', 'README.md')).read()
+
+MIN_PYTHON_VERSION = '>=3.6.*'
+
+requirements = [
+    'Pillow<=6.2.1',
+    'numpy>=1.16.0',
```

Comment:
here there is min numpy version, but in the other install-script there is not

Reply:
I double checked, it's there:
https://github.com/gradientsky/autogluon/blob/modules-v2/core/setup.py#L44
https://github.com/gradientsky/autogluon/blob/modules-v2/tabular/setup.py#L42
https://github.com/gradientsky/autogluon/blob/modules-v2/extra/setup.py#L44
https://github.com/gradientsky/autogluon/blob/modules-v2/mxnet/setup.py#L42
https://github.com/gradientsky/autogluon/blob/modules-v2/text/setup.py#L44
https://github.com/gradientsky/autogluon/blob/modules-v2/vision/setup.py#L44